### PR TITLE
SLAP-2502: Implement Feedback Widget To Specific Actions On The Manager User

### PR DESCRIPTION
**Pull request by maddy531**
Resolves [SLAP-2502](https://jira.jstor.org/browse/SLAP-2502)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR adds our Feedback Widget in order to generate user satisfaction baseline.

* **What is the current behavior?** (You can also link to an open issue here)
When you click `Export` , `Save` (after editing a user), or `Register` (after registering a new user) on the Manager Users page, a Feedback Widget popup should appear.

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by StevenACoffman** [comment]
Looks good to me!

File: src/app/users-page/users-page.component.ts [hunk 1]
```diff
@@ -63,6 +67,14 @@ export class UsersPage implements OnInit, OnDestroy {
   }
 
   ngOnInit():void {
+
+    // Script needs to be initiated on init in order to run the the Feedback Widget
+    const s = this.renderer2.createElement('script');
+    s.type = 'text/javascript';
+    s.src = 'http://localhost:4200/fetch-widget/feedback.js';
```
Comment (maddy531) [inline src/app/users-page/users-page.component.ts]: Needs to be updated to dynamically toggle between EVS

File: src/app/modals/register/register-modal.component.pug [hunk 2]
```diff
@@ -40,5 +40,10 @@ form#formRegisterUsers([formGroup]="registerForm", (ngSubmit)="onSubmit(register
       type="button"
     ) {{ 'REGISTER_MODAL.BUTTONS.CLOSE' | translate }}
     button#btnPermissionsUpdate.btn.btn-primary(
+      data-nsfw="Register New User"
+      attr.data-user-id="{{adminUser.default_user_id}}-id"
+      attr.data-session-id="{{adminUser.id}}-session"
```
Comment (maddy531) [inline src/app/modals/register/register-modal.component.pug]: "123-session"
user-id - needs to be in UUID format.

File: src/app/modals/register/register-modal.component.pug [hunk 3]
```diff
@@ -40,5 +40,10 @@ form#formRegisterUsers([formGroup]="registerForm", (ngSubmit)="onSubmit(register
...
+      attr.data-site-code="{{adminUser.id}}-code"
```
Comment (maddy531) [inline src/app/modals/register/register-modal.component.pug]: remove `-code`

**Pull request merged**